templates/includes/navbar.html

```diff
@@ -0,0 +1,31 @@
+<nav class="navbar navbar-expand-lg navbar-light bg-light" style="color: black">
+  <a class="navbar-brand" href="{% url 'blog:index' %}">BLOG WEBSITE</a>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav"
+          aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <div class="collapse navbar-collapse" id="navbarNav">
+    <ul class="navbar-nav">
+      {% if not user.is_authenticated %}
+        <li class="nav-item active">
+          <a class="nav-link" href="{% url 'login' %}">Log In</a>
+        </li>
+        <li class="disabled">
+          <a class="nav-link" href="{% url 'logout' %}">Log Out</a>
+        </li>
+      {% else %}
+        <li class="disabled">
+          <a class="nav-link" href="{% url 'login' %}">Log In</a>
+        </li>
+        <li class="nav-item active">
+          <a class="nav-link" href="{% url 'logout' %}">Log Out</a>
+        </li>
+      {% endif %}
```

Comment:
Code style issue: Same as above, indent the code inside the `{% else %}` and `{% endif %}` tags to make it more readable.